changelog/18.0/18.0.0/summary.md

```diff
@@ -34,6 +34,7 @@
     - [Debian: Buster removed](#debian-buster)
   - **[Durability Policies](#durability-policies)**
     - [New Durability Policies](#new-durability-policies)
+  - **[Experimental Foreign Key Support](#foreign-keys)**
```

Comment:
I feel like this should be moved up a little, it's a major item of `v18.0.0`

Reply:
Okay 👍